lib/aca_entities/fdsh/ridp/h139/contracts/secondary_request_contract.rb

```diff
@@ -8,6 +8,7 @@ module H139
         class SecondaryRequestContract < Dry::Validation::Contract
           params do
             required(:SessionIdentification).filled(:string)
+            required(:DSHReferenceNumber).filled(:string)
```

Comment:
@raghuramg Why are you making this required instead of optional? If this feature is being developed using the Resource Registry feature flag, how can we conditionally not require this field when the feature is disabled?

Reply:
updated h139 xsd as well to add this element under SecondaryRequest